Assets/Scripts/Scenes/InGame/System/UI/EscapeItemPresenter.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UniRx;
+using Scenes.Ingame.Manager;
+using TMPro;
+using UnityEngine.UI;
+
+namespace Scenes.Ingame.InGameSystem.UI
+{
+
+    public class EscapeItemPresenter : MonoBehaviour
+    {
+        [SerializeField, Tooltip("�E�o�A�C�e������\��")]
+        TextMeshProUGUI _socreText;
+        [SerializeField, Tooltip("Ingame")]
+        GameObject ingameManager;
+
+        void Start()
+        {
+            IngameManager ingamemanager = ingameManager.GetComponent<IngameManager>();
+            ingamemanager.OnEscapeCount.Subscribe(x => {
+                _socreText.text = "�E�o�A�C�e����: " + x;
+            }).AddTo(this);
+        }
```

Comment:
```suggestion

        void Start()
        {
            IngameManager ingamemanager = IngameManager.Instance;
            ingamemanager.OnEscapeCount.Subscribe(x => {
                _socreText.text = "�E�o�A�C�e����: " + x;
            }).AddTo(this);
        }
```
IngameManagerはシングルトンで設計されているので、上記のようにすると簡単に取得できます。